### DISCUSSION TITLE
能提供对文件夹内文件的默认排序方式和搜索功能吗

### DISCUSSION THREAD

**Discussion by roiiiu** [AlistGo/alist#3702] [Ideas]
挂在云盘文件经常乱序，想要能提供个默认首字母顺序排序和搜索文件功能。

**Comment by Hunter-Source** [comment]
在挂载网盘的设置里面有相关的排序设置，可以尝试一下，看能不能解决问题。
![image](https://github.com/alist-org/alist/assets/77777504/5324b1e8-2a3f-4124-b6fe-f68058fd797c)
![image](https://github.com/alist-org/alist/assets/77777504/a9a6155c-22e0-47a4-a36c-d9c3dc2a40a5)
![image](https://github.com/alist-org/alist/assets/77777504/aff4208c-fd7e-44d5-997c-fb21f29c532a)

**Reply by WangTangJang** [reply]
我的可以解决，感谢提示。

**Reply by fgr178707** [reply]
> 在挂载网盘的设置里面有相关的排序设置，可以尝试一下，看能不能解决问题。 ![image](https://private-user-images.githubusercontent.com/77777504/296543138-5324b1e8-2a3f-4124-b6fe-f68058fd797c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3OTk0MTEsIm5iZiI6MTcyMzc5OTExMSwicGF0aCI6Ii83Nzc3NzUwNC8yOTY1NDMxMzgtNTMyNGIxZTgtMmEzZi00MTI0LWI2ZmUtZjY4MDU4ZmQ3OTdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE2VDA5MDUxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0YTc4NmZiZjM1ODUyODkxZWQ4NDJiY2E4NGZhZTUwOWYzYzgxNjY5ZmNkYzgxNjM5MjMyMzUyODk5ZGRmNWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bDdt8TotmLzcStHvmQK3UxfIxeSiwOooOrjqAYX0SMM) ![image](https://private-user-images.githubusercontent.com/77777504/296543153-a9a6155c-22e0-47a4-a36c-d9c3dc2a40a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3OTk0MTEsIm5iZiI6MTcyMzc5OTExMSwicGF0aCI6Ii83Nzc3NzUwNC8yOTY1NDMxNTMtYTlhNjE1NWMtMjJlMC00N2E0LWEzNmMtZDljM2RjMmE0MGE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE2VDA5MDUxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViYTIyZDYyZThlZTlmOWVjNzFhM2VhZDY3ODVhYzlhNjA2MzMyYzI2ODFiODM2MzI3N2RlMmI5ODFjZWI3Y2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.K3M973hUUypWnL-YeDM34PdLYl78ivEEZ9z4C4bTyuY) ![image](https://private-user-images.githubusercontent.com/77777504/296543162-aff4208c-fd7e-44d5-997c-fb21f29c532a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3OTk0MTEsIm5iZiI6MTcyMzc5OTExMSwicGF0aCI6Ii83Nzc3NzUwNC8yOTY1NDMxNjItYWZmNDIwOGMtZmQ3ZS00NGQ1LTk5N2MtZmIyMWYyOWM1MzJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE2VDA5MDUxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4MjI1Yzc5ZmI2MzMxMjlmZWYyOTI0ODhhMjZmYTc3MTUxZDQ0YzYwYzVjNDEyODY1YjYwYTgzODRmZDY4YWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eyb0f8wr_8CKKVyd7TlyKpP_APC7wYgsbUU9vj4XpMM)

原来115网盘的排序是跟着官网走的
我说呢，在设置里面一直找不着115的文件夹排序方式
